### PR TITLE
Feature/fs group doc

### PR DESCRIPTION
**Pull request by harishp8889**
# Description
FSGroup policy known issue update

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/167 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by shanmydell — APPROVED** [review]

File: content/docs/csidriver/release/powerstore.md [hunk 1]
```diff
@@ -8,16 +8,18 @@ description: Release notes for PowerStore CSI driver
...
+| Delete namespace that has PVCs and pods created with the driver. The External health monitor sidecar crashes as a result of this operation | Deleting the namespace deletes the PVCs first and then removes the pods in the namespace. This brings a condition where pods exist without their PVCs and causes the external-health-monitor sidecar to crash. This is a known issue and has been reported at https://github.com/kubernetes-csi/external-health-monitor/issues/100 <br> |
+| FSGroupPolicy may not work as expected without root privileges for NFS only<br/>https://github.com/kubernetes/examples/issues/260          | To get the desired behavior set "allowRoot: "true" in the storage class parameter                                                                                                                                                                                                                                                       |
```
Comment (randeepsharma) [inline content/docs/csidriver/release/powerstore.md:22]: Better to keep fsGroupPolicy in same case everywhere.

**Review by shanmydell — DISMISSED** [review]

File: content/docs/csidriver/release/powerstore.md (hunk 1 above)
Comment (prablr79) [reply content/docs/csidriver/release/powerstore.md:22]: I agree

**Review by spriya-m — APPROVED** [review]

File: content/docs/csidriver/release/powerstore.md (hunk 1 above)
Comment (harishp8889) [reply content/docs/csidriver/release/powerstore.md]: fixed

**Review by prablr79 — DISMISSED** [review]

File: content/docs/csidriver/release/powerscale.md [hunk 2]
```diff
@@ -8,20 +8,20 @@ description: Release notes for PowerScale CSI driver
...
+|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```
Comment (randeepsharma) [inline content/docs/csidriver/release/powerscale.md:20]: Not sure if this change is needed,

**Review by randeepsharma — APPROVED** [review]

**Review by spriya-m — APPROVED** [review]

File: content/docs/csidriver/release/powerscale.md (hunk 2 above)
Comment (harishp8889) [reply content/docs/csidriver/release/powerscale.md:20]: This is a formatting issue fix, generated automatically and not breaking anything

**Pull request merged**